pkg/memtier/prompt_test.go

```diff
@@ -42,6 +42,9 @@ func FuzzPrompt(f *testing.F) {
 	pagesCommonArgs := fmt.Sprintf(" -pid %d ", os.Getpid())
 	testcases := []string{
 		"help",
+		"log -i \"my info\" -d 'my debug' -e my\\ error",
+		"log --copy --prefix foo",
+		"log --capture x",
```

Comment:
optCapture := p.f.Bool("capture", false, "redirect all prompt output to log")
capture should be a bool, is "x" needed here?

Reply:
Good catch, I'll fix it

Reply:
Fixed.